### PR TITLE
Relocating favicons

### PR DESCRIPTION
**Pull request by gvwilson**
This moves all the favicon files _except_ `favicon.ico` into `assets/img`, with corresponding changes to `_includes/header`, and removes the now-unused `_includes/favicon`. @todrobbins and @pbanaszkiewicz, please review.

### CODE REVIEW

File: _includes/head [hunk 1]
```diff
@@ -25,22 +25,22 @@
...
-	<link rel="apple-touch-icon" sizes="152x152" href="/apple-touch-icon-152x152.png">
-	<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon-180x180.png">
-	<link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
-	<link rel="icon" type="image/png" href="/android-chrome-192x192.png" sizes="192x192">
-	<link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96">
-	<link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
-	<link rel="manifest" href="/manifest.json">
-	<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#293c92">
-	<link rel="shortcut icon" href="/favicon.ico">
+	<link rel="shortcut icon" href="/favicon.ico"> <!-- must remain in root directory -->
```
Comment (pbanaszkiewicz) [inline _includes/head:28]: If other `<link>`s have `{{ site.url }}`, so why doesn't this one have it?

**Comment by pbanaszkiewicz** [comment]
@gvwilson you have to change contents of `manifest.json`, too.

**Comment by gvwilson** [comment]
manifest.json and browserconfig.xml updated.

**Comment by todrobbins** [comment]
Looks good. Merge baby! Merge!

**Pull request merged**